src/main/java/com/formulasearchengine/mathmlconverters/MathMLConverter.java

```diff
@@ -98,7 +103,8 @@ public String transform(Element formulaNode) throws Exception, MathConverterExce
                 try {
                     Element applyNode = (Element) XMLHelper.getElementB(formulaNode, xPath.compile("//m:apply"));
                     formulaId = applyNode.getAttribute("id");
-                } catch (Exception e) { }
+                } catch (Exception e) {
+                }
```

Comment:
Add a log information here. At least on trace-level (if an exception on this position is "allowed"). If this exception is not allowed the complete exception handling is missing.